eng/run-test-job.yml

```diff
@@ -355,3 +355,12 @@ jobs:
           scenarios:
           - jitelthookenabled
           - jitelthookenabled_tiered
+
+    # Publish Logs
+    - task: PublishBuildArtifacts@1
+      displayName: Publish Logs
+      inputs:
+        pathtoPublish: $(Build.SourcesDirectory)/bin/Logs
+        artifactName: ${{ format('TestLogs_run_test_job_{0}{1}_{2}_{3}_{4}_{5}_{6}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig, parameters.managedTestBuildOsGroup, parameters.managedTestBuildOsSubgroup, parameters.testGroup) }}
```

Comment:
TestRunLogs_ as the prefix? Tiny bit shorter...

Reply:
ok